src/controllers/authController.js

```diff
@@ -0,0 +1,30 @@
+import models from "../database/models/index.js";
+const {Users} = models;
+import helper from "../utils/jwtHelper.js";
+const { comparePassword } = require("../utils");
```

Comment:
```suggestion
import { comparePassword } from "../utils/index.js";
```
require is es5. We must use es6 as much as possible